README.md

```diff
@@ -2,3 +2,4 @@
 for testing out git
 
 [Sohan Rudra](https://github.com/rudrasohan)
+[Shrey Shrivastava](https://github.com/shreyshrivastava4799)
```

Comment:
@shreyshrivastava4799 please see the changes requested in #1 and update the PR accordingly.